wd_capabilities.au3

```diff
@@ -281,14 +261,14 @@ EndFunc   ;==>_WD_CapabilitiesGet
 ; Name ..........: _WD_CapabilitiesDefine
 ; Description ...: Define new capability type and name
 ; Syntax ........: _WD_CapabilitiesDefine(Byref $sCapabilityType, $sCapabilityName)
-; Parameters ....: $sCapabilityType - reference to $_WD_KEYS__* value that should be suplemented for supporting new capability name
-;                  $sCapabilityName  - Name of new capability that should be supported
+; Parameters ....: $sCapabilityType     - reference to $_WD_KEYS__* value that should be suplemented for supporting new capability name
+;                  $sCapabilityName     - Name of new capability that should be supported
 ; Return values .: Success - none.
 ;                  Failure - none and sets @error to one of the following values:
 ;                  - $_WD_ERROR_InvalidDataType
 ;                  - $_WD_ERROR_InvalidValue
 ;                  - $_WD_ERROR_NotSupported
-;                  - $_WD_ERROR_InvalidArgue
+;                  - $_WD_ERROR_AlreadyDefined
```

Comment:
Guessing your PR is incomplete since this doesn't yet exist

Reply:
I will this suplement soon

Reply:
Take a look here:

[core: $_WD_ERROR_AlreadyDefined](https://github.com/Danp2/au3WebDriver/pull/285/commits/c855bd81aefcb063eb1e715377a34e4bc06dc389)

Reply:
Don't forget that `$aWD_ERROR_DESC` needs to be updated as well

Reply:
[Capability or value already defined](https://github.com/Danp2/au3WebDriver/pull/285/commits/7cb2e943ec01cafb0fe9bde2d0ae2f8b4661ecc0)